homeassistant/components/fritzbox/sensor.py

```diff
@@ -6,30 +6,25 @@
 from datetime import datetime
 from typing import Final
 
-from pyfritzhome.fritzhomedevice import FritzhomeDevice
-
 from homeassistant.components.climate.const import PRESET_COMFORT, PRESET_ECO
-from homeassistant.components.sensor import (
-    SensorDeviceClass,
-    SensorEntity,
-    SensorEntityDescription,
-    SensorStateClass,
-)
+from homeassistant.components.sensor import (SensorDeviceClass, SensorEntity,
+                                             SensorEntityDescription,
+                                             SensorStateClass)
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import (
-    ENERGY_KILO_WATT_HOUR,
-    PERCENTAGE,
-    POWER_WATT,
-    TEMP_CELSIUS,
-)
+from homeassistant.const import (ELECTRIC_CURRENT_AMPERE,
+                                 ELECTRIC_POTENTIAL_VOLT,
+                                 ENERGY_KILO_WATT_HOUR, PERCENTAGE, POWER_WATT,
+                                 TEMP_CELSIUS)
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity import EntityCategory
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import StateType
 from homeassistant.util.dt import utc_from_timestamp
+from pyfritzhome.fritzhomedevice import FritzhomeDevice
 
 from . import FritzBoxEntity
-from .const import CONF_COORDINATOR, DOMAIN as FRITZBOX_DOMAIN
+from .const import CONF_COORDINATOR
+from .const import DOMAIN as FRITZBOX_DOMAIN
```

Comment:
> ... or at least make sure to follow our [style guidelines](https://developers.home-assistant.io/docs/development_guidelines)

with this I mean the [sorting of the imports](https://developers.home-assistant.io/docs/development_guidelines/#ordering-of-imports) and formatting of the code by using Black (_`black --fast homeassistant`_)

Reply:
Hopefully fixed now